### PR TITLE
EIA: Preliminary support for INTL, SEDS, TOTAL

### PR DESCRIPTION
**Pull request by pradh**
Include schema-less support for 3 more datasets.

Notable exceptions:

* INTL: excludes places that we don't have entities for, like EU, OECD, Eurasia, etc.
* TOTAL: excludes non-US countries with import/export stats, because the 2-letter code is not ISO/FIPS compatible
* SEDS: excludes 2 Federal offshore places that we don't have entities for.

Also included:

* Quarterly format test-case in ELEC
* Add download_bulk.py args to specify list of datasets

### CODE REVIEW

File: scripts/us_eia/opendata/process/common.py [hunk 1]
```diff
@@ -89,10 +95,28 @@ def _find_dc_place(raw_place, is_us_place, counters):
...
             return alpha2_to_dcid.USSTATE_MAP[raw_place]
     else:
-        if raw_place in alpha2_to_dcid.COUNTRY_MAP:
+        if len(raw_place) == 2 and raw_place in alpha2_to_dcid.COUNTRY_MAP:
             return alpha2_to_dcid.COUNTRY_MAP[raw_place]
+        elif len(raw_place) == 3 and raw_place in _ALPHA3_COUNTRY_SET:
+            return f'country/{raw_place}'
+        elif len(raw_place) > 3:
+            # INTL dataset has 40 country aggregates
+            # (http://screenshot/8YvH4w58iDtuc5r).  We map a subset that exists
```
Comment (beets) [inline scripts/us_eia/opendata/process/common.py:104]: maybe find a way for github to host this?  this is a bit of a hack but would work:
https://medium.com/@justynagolawska/how-to-easily-add-screenshots-into-your-readme-file-on-github-d806a01d6ffd

File: scripts/us_eia/opendata/process/seds.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        series_id: EIA series ID
+        counters: map for updating error statistics
+
+    Returns a (place, raw-stat-var, is_us_place) tuple.
+    """
+
+    # SEDS.{MEASURE}.{PLACE}.{PERIOD}
+    #
+    # NOTE: This pattern excludes 11 series_ids which deal with two Federal
+    # Offshore places (http://screenshot/B2LNQ9tKG4Smcbd).
```
Comment (beets) [inline scripts/us_eia/opendata/process/seds.py:19]: ditto here

**Review by beets — APPROVED** [review]
wow nice

File: scripts/us_eia/opendata/process/common.py (hunk 1 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/common.py]: good catch. nice find.

File: scripts/us_eia/opendata/process/seds.py (hunk 2 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/seds.py]: Done.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**